include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -422,7 +422,7 @@ struct __stable_sort_fn
     {
         const auto __dispatch_tag = oneapi::dpl::__ranges::__select_backend(__exec);
         return oneapi::dpl::__internal::__ranges::__pattern_sort_ranges(__dispatch_tag,
-            std::forward<_ExecutionPolicy>(__exec), std::forward<_R>(__r), __comp, __proj);
+            std::forward<_ExecutionPolicy>(__exec), std::forward<_R>(__r), __comp, __proj, std::ranges::stable_sort);
```

Comment:
Could you include <algorithm>?
This line is also too long. I suggest applying clang-format.

Reply:
1) done
2) 118 symbols; The limitation is 120